src/views/MarketFilter.tsx

```diff
@@ -34,28 +35,25 @@ export const MarketFilter = ({
 
   return (
     <Styled.MarketFilter>
-        <SearchInput
-          placeholder={stringGetter({ key: STRING_KEYS.MARKET_SEARCH_PLACEHOLDER })}
-          onTextChange={onSearchTextChange}
+      <Styled.ToggleGroupContainer>
+        <ToggleGroup
+          items={Object.values(filters).map((value) => ({
+            label: stringGetter({ key: MARKET_FILTER_LABELS[value] }),
+            value,
+          }))}
+          value={selectedFilter}
+          onValueChange={onChangeFilter}
         />
-        <Styled.ToggleGroupContainer>
-          <ToggleGroup
-            items={Object.values(filters).map((value) => ({
-              label: stringGetter({ key: MARKET_FILTER_LABELS[value] }),
-              value,
-            }))}
-            value={selectedFilter}
-            onValueChange={onChangeFilter}
-          />
-          {hasPotentialMarketsData && !hideNewMarketButton && (
-            <Button
-              onClick={() => navigate(`${AppRoute.Markets}/${MarketsRoute.New}`)}
-              size={ButtonSize.Small}
-            >
-              {stringGetter({ key: STRING_KEYS.PROPOSE_NEW_MARKET })}
-            </Button>
-          )}
-        </Styled.ToggleGroupContainer>
+        {hasPotentialMarketsData && !hideNewMarketButton && (
+          <Button
+            onClick={() => navigate(`${AppRoute.Markets}/${MarketsRoute.New}`)}
+            size={ButtonSize.Small}
+          >
+            {stringGetter({ key: STRING_KEYS.PROPOSE_NEW_MARKET })}
+          </Button>
+        )}
+      </Styled.ToggleGroupContainer>
+      <SearchInput placeholder={'Search'} onTextChange={onSearchTextChange} />
```

Comment:
The `MarketFilter` component is also used on the trade page. The changes here affect the other component as well. Should use props in order to get the desired layout changes.

| current | new |
| --- | --- |
| <img width="709" alt="Screen Shot 2024-04-29 at 11 39 15 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/f768bd25-11e0-4391-95e7-c8d322d77009"> | <img width="710" alt="Screen Shot 2024-04-29 at 11 39 02 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/644e4e96-dca1-4fbe-b7b3-ed71f4b3b5e3"> |

Reply:
Use `stringGetter` for `Search` placeholder

Reply:
Oh ye, sorry I missed it, I'll fix it now!

Reply:
Done! I also add the stringGetter